Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionTouchable.cs

```diff
@@ -218,10 +218,13 @@ public virtual float DistanceToSurface(Vector3 samplePoint)
                 Vector3.Dot(pointOnPlane, LocalRight),
                 Vector3.Dot(pointOnPlane, localUp));
 
-            // Clamp to bounds
-            planeSpacePoint = new Vector2(
-                Mathf.Clamp(planeSpacePoint.x, -bounds.x / 2, bounds.x / 2),
-                Mathf.Clamp(planeSpacePoint.y, -bounds.y / 2, bounds.y / 2));
+            if (planeSpacePoint.x < -bounds.x / 2 ||
```

Comment:
Could you please add a comment here explaining why we are doing this in the code, so that when somebody new to the code reads it they know why we did it?

Reply:
I want to give the code a second look. If we are doing things this way there's a better way to do it. But I'll make sure to explain reasoning.